src/python/platforms/fuchsia/util/fuzzer.py

```diff
@@ -351,11 +351,15 @@ def merge(self, fuzzer_args):
       fuzzer_args.append('data/corpus/')
       fuzzer_args.append('data/corpus.prev/')
 
-    # Save mergefile in case we are interrupted
-    fuzzer_args = ['-merge=1', '-merge_control_file=data/.mergefile'
-                  ] + fuzzer_args
+    merge_args = ['-merge=1']
```

Comment:
@flowerhack @eepeep  FYI for changes here. Was this completely broken before? 

@Dor1s can you please make sure that existing tests pass with:

```
FUCHSIA_TESTS=1 python butler.py py_unittest -p '*engine*' 
```

Reply:
Believe it or not, all passed. Thanks for providing me with these commands!